views/lib/loggedin.hbs

```diff
@@ -74,6 +74,11 @@
                 <i class="fa fa-bars"></i>
             </a>
             <div class="content-min-height">
+                <section class="demo" id="offlineAlert" style="display:none;">
```

Comment:
I can't get this to work/show locally. Might doin' it wrong

Reply:
switch offline in developer tools or disable wifi should display that message